src/llamacrawl/services/utils/console.py

```
@@ -118,14 +118,21 @@ def __init__(self, description: str, total: int | None) -> None:
 class Progress:
     """Lightweight drop-in replacement for rich.progress.Progress."""
 
-    def __init__(self, *_: Any, console: Console | None = None, transient: bool | None = None) -> None:
+    def __init__(
+        self, *_: Any, console: Console | None = None, transient: bool | None = None
```

Ruff (0.14.0)

Dynamically typed expressions (typing.Any) are disallowed in `*_`

(ANN401)